internal/cli/login.go

```diff
@@ -153,11 +154,55 @@ func (o *LoginOptions) Validate(args []string) error {
 	if err != nil {
 		return fmt.Errorf("API URL is not a valid URL: %w", err)
 	}
+
+	// Check for missing protocol first for clearer guidance
+	if !strings.HasPrefix(args[0], "http") {
+		return fmt.Errorf("API URL is missing the protocol. Please ensure the API URL starts with 'https://'")
+	}
+
+	// Enforce HTTPS scheme
 	if parsedUrl.Scheme != "https" {
 		return fmt.Errorf("the API URL must use HTTPS for secure communication. Please ensure the API URL starts with 'https://' and try again")
 	}
+
+	// Check for valid host
 	if parsedUrl.Host == "" {
-		return fmt.Errorf("API URL is not a valid URL")
+		return fmt.Errorf("API URL is missing a valid hostname. Please provide a complete URL with hostname")
+	}
+
+	// Check for extra path components that might indicate user error
+	if parsedUrl.Path != "" && parsedUrl.Path != "/" {
```

Comment:
It's entirely possible in some deployments that it might be setup under a path (e.g. going through a proxy). It is far too early in the life of this project to know that it won't ever be.